gapis/client/client.go

```diff
@@ -329,3 +329,79 @@ func (c *client) ClientEvent(ctx context.Context, req *service.ClientEventReques
 	_, err := c.client.ClientEvent(ctx, req)
 	return err
 }
+
+// TraceTargetTreeNode returns a node in the trace target tree for the given device
+func (c *client) TraceTargetTreeNode(ctx context.Context, req *service.TraceTargetTreeRequest) (*service.TraceTargetTreeNode, error) {
+	res, err := c.client.TraceTargetTreeNode(ctx, req)
+
```

Comment:
nit: try to keep the error handling close to the source (remove the newline)

Reply:
Done